### PR TITLE
Add Route library (Router #1838)

### LINKED ISSUE #1838: Router
[Router](https://github.com/freshOS/Router) - 🛣 Simple Navigation for iOS :large_orange_diamond:

### PR DESCRIPTION
**Pull request by serhii-londar**
Rename **URL Scheme** section to **App Routing**
Add Router lib (close #1838)
Move [Marshroute](https://github.com/avito-tech/Marshroute) lib to **App Routing** section
Move [SwiftRouter](https://github.com/skyline75489/SwiftRouter) to **App Routing** section

### CODE REVIEW

**Comment by danger-awesome-ios** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          2 Warnings
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Too many changes (when adding, please keep it to one project per Pull Request)</td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Found 3 link issues, a project collaborator will take care of these, thanks :)</td>
    </tr>
  </tbody>
</table>

#### Link issues by [`awesome_bot`](https://github.com/dkhamsing/awesome_bot)

  Line | Status | Link
| ---: | :----: | --- |
854 | [301](https://httpstatuses.com/301) | https://github.com/terflogag/FacebookImagePicker redirects to<br>https://github.com/floriangbh/FacebookImagePicker
1355 | [301](https://httpstatuses.com/301) | https://github.com/SwiftOnEdge/Edge redirects to<br>https://github.com/skylab-inc/Edge
2388 | [301](https://httpstatuses.com/301) | https://github.com/terflogag/OpenSourceController redirects to<br>https://github.com/floriangbh/OpenSourceController


<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

**Comment by lfarah** [comment]
Thank you for these issue fixes. Please also add this to the contents section

**Comment by serhii-londar** [comment]
Update contents section.

**Comment by serhii-londar** [comment]
Moved App Routing section alphabetically.

File: README.md [hunk 1]
```diff
@@ -2321,7 +2320,7 @@ Most of these are paid services, some have free tiers.
 * [SVWebViewController](https://github.com/TransitApp/SVWebViewController) - A drop-in inline browser for your iOS app.
 * [PTPopupWebView](https://github.com/pjocprac/PTPopupWebView) - PTPopupWebView is a simple and useful WebView for iOS, which can be popup and has many of the customized item. :large_orange_diamond:
 
-## URL Scheme
+## App Routing
```
Comment (dkhamsing) [inline README.md]: Thanks for this, could you move the whole App Routing content under Analytics

Comment (serhii-londar) [reply README.md]: Fixed in new commit.

**Comment by dkhamsing** [comment]
Thank you!

**Pull request merged**